src/components/managed-street.js

```diff
@@ -54,6 +55,8 @@ AFRAME.registerComponent('managed-street', {
     }
   },
   init: function () {
+    this.zustandState = useStore.getState();
```

Comment:
Don't do this, you get a snapshot of the state and it may be stale when you'll then use that, checking an old state instead of the current one. Use `const zustandState = useStore.getState()` just before you use it.

Reply:
086c2eaa42e65246af9ea69f206234cf14cd9c91